File: Modul_4/Bahasa_C/tempCodeRunnerFile.c
#include <stdio.h>
int main() {
    
    int nilai, kelipatan;
    
    printf("Masukkan Nilai: ");
    scanf ("%d", &nilai);
    printf("Masukkan Kelipatan: ");
    scanf ("%d", &kelipatan);

    int total = 0;

    for (int i = 1; i <= nilai; i++){
        int hasil = 0;

        for (int j=i; j>= 1; j--) {
            printf ("(%d * %d)", j, kelipatan);
            hasil += (j * kelipatan);
            if (j != 1) printf(" + ");
        }
        printf(" = %d\n", hasil);
        total += hasil;

    }
     printf("%d\n", total);
     return 0;
}

File: Modul_4/Bahasa_Python/tempCodeRunnerFile.py
nilai=int(input ("Masukkan Nilai: "))
kelipatan =int(input ("Masukkan Kelipatan: "))
total1 = 0

for i in range (1, nilai + 1):
    total2 = 0
    rumus= ""
    for j in range (i, 0, -1):
        hasil = j * kelipatan
        total2 += hasil
        rumus += f"({j} * {kelipatan})"
        if j != 1:
            rumus += " + "
    
    print(f"{rumus} = {total2}")
    total1 += total2

print(total1)